setup.py

```diff
@@ -118,10 +117,17 @@
     'postgresql': [
         'invenio-db[postgresql,versioning]>=1.0.0b2',
     ],
+    'build-node': [
+        'ipdb',
```

Comment:
Have you tried `pdbpp` ?

Reply:
It's also very nice indeed. I've put `ipdb` because that's what was forced in via puppet before. Would you recommend it instead of ipdb?

Reply:
the advantage of pdbpp was that has a 'published' pytest plugin (that you can actually just pip install) right?
Though I'm not used to it :/ (it's also a different shell than the one provided by `inspirehep` app).

Reply:
I like it better because they have a more pragmatic approach than `ipdb`: they don't have to wait for IPython to implement a feature in order to have it, they just do it on their own. But it's not a blocker here!

Reply:
Actually we can simply have both :+1:

Reply:
And what about: https://pypi.python.org/pypi/pudb? Brings back so many memories.